master/build.xml

```diff
@@ -90,7 +90,7 @@
         <delete dir="${build}" />
         <delete dir="${src}" />
         <delete dir="${meta-inf}" />
-        <ant dir="${example}" target="clean" inheritAll="false"/>
+        <!--<ant dir="${example}" target="clean" inheritAll="false"/>-->
```

Comment:
Please remove commented out code.

Reply:
Comments Incorporated.

Reply:
Yes, this is a ANT based project and we need some specific jars for the build to work. So, we are checking in these jars.